packages/@lwc/babel-plugin-component/src/decorators/api/validate.ts

```diff
@@ -30,27 +33,27 @@ function validateConflict(path, decorators) {
     }
 }
 
-function isBooleanPropDefaultTrue(property) {
-    const propertyValue = property.node.value;
+function isBooleanPropDefaultTrue(property: NodePath<types.Node>) {
```

Comment:
```suggestion
function isBooleanPropDefaultTrue(property: NodePath<types.ObjectProperty>) {
```

Reply:
This actually breaks the types because it's expecting a `ClassMethod` elsewhere.